ramls/delayedTasks.raml

```diff
@@ -43,3 +43,9 @@ traits:
           body:
             text/plain:
               example: "Internal server error"
+  /retryFailedEmails:
+    post:
```

Comment:
is: [validate] ?

Reply:
there is nothing to validate, request has no body